modules/custom/openy_home_branch/js/hb_cookies_storage.js

```diff
@@ -80,7 +80,7 @@
     // Move current data to Cookies storage.
     updateStorage: function () {
       $(document).trigger('hb-before-storage-update', this.data);
-      $.cookie('home_branch', JSON.stringify(this.data), { expires: 360, path: '/' });
+      $.cookie('home_branch', JSON.stringify(this.data), { expires: 360, path: drupalSettings.path.baseUrl });
```

Comment:
thank you!!!!!!!

Reply:
@podarok no problem)

Our team interested in the best solution, which will suit everyone, so thank you for your feedback and support, now Home Branch works much better